sentry-android-core/src/main/java/io/sentry/android/core/ActivityFramesTracker.java

```diff
@@ -25,8 +26,10 @@
   private final @NotNull Map<SentryId, Map<String, @NotNull MeasurementValue>>
       activityMeasurements = new ConcurrentHashMap<>();
 
-  public ActivityFramesTracker(final @NotNull LoadClass loadClass) {
-    androidXAvailable = checkAndroidXAvailability(loadClass);
+  public ActivityFramesTracker(
+      final @NotNull LoadClass loadClass, final @NotNull SentryOptions options) {
```

Comment:
This would mean that we can't create this class before initing the SDK which gives us the options, can we pass the logger only?

Reply:
Yes, I'd make it nullable with an extra ctor, but I'd still move to a Logger only, Hybrid SDKs have their own logger, but passing an option sounds wrong in this case.